doc/src/pair_lj_sphere.rst

```diff
@@ -0,0 +1,158 @@
+.. index:: pair_style lj/sphere
+.. index:: pair_style lj/sphere/omp
+
+pair_style lj/sphere command
+============================
+
+Accelerator Variant: *lj/sphere/omp*
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   pair_style style args
+
+* style = *lj/sphere*
+* args = list of arguments for a particular style
+
+.. parsed-literal::
+
+     *lj/sphere* args = cutoff ratio
+       cutoff = global cutoff ratio for Lennard Jones interactions (unitless)
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   pair_style lj/sphere 2.5
+   pair_coeff * * 1.0
+   pair_coeff 1 1 1.1 2.8
+
+Description
+"""""""""""
+
+The *lj/sphere* style compute the standard 12/6 Lennard-Jones potential,
+given by
+
+.. math::
+
+   E = 4 \epsilon \left[ \left(\frac{\sigma_{ij}}{r}\right)^{12} -
+       \left(\frac{\sigma_{ij}}{r}\right)^6 \right]  \qquad  r < r_c * \sigma_{ij}
+
+:math:`r_c` is the cutoff ratio.
+
+This is the same potential function as used by the :doc:`lj/cut
+<pair_lj>` pair style, but the :math:`\sigma_{ij}` parameter is not set
+as a per-type parameter via the :doc:`pair_coeff command <pair_coeff>`,
+but taken from the per-atom diameter attribute of :doc:`atom_style
+sphere <atom_style>`.  The individual value of :math:`\sigma_{ij}` is
+computed from the diameters of the two atoms using the mixing rule for
+pair coefficients as set by the :doc:`pair_modify mix <pair_modify>`
+command (defaults to geometric mixing).  The cutoff is not specified as
+a distance, but as ratio that is internally multiplied with
+:math:`\sigma_{ij}` to obtain the actual cutoff for each pair of atoms.
+
+Note that :math:`\sigma_{ij}` is defined in the LJ formula above as the
+zero-crossing distance for the potential, *not* as the energy minimum which
+is at :math:`2^{\frac{1}{6}} \sigma_{ij}`.
+
+.. admonition:: Notes on cutoffs, neighbor lists, and efficiency
+   :class: note
+
+   Because the cutoff in this pair style depends on the diameter of the
+   atoms, this influences which cutoff is used to build neighbor lists
+   and how effective those neighbor lists are in avoiding to compute
+   distances between non-interacting atoms.  This pair style uses a
+   conventional neighbor list construction like for :doc:`pair style
+   lj/cut <pair_lj>`, where the cutoffs are typically rather similar.
+   LAMMPS will determine the largest cutoff and use this value for
+   building the neighbor lists.  This can be inefficient, if the
+   difference between cutoffs is very large.  The command :doc:`neighbor
+   multi <neighbor>` can be used to enable a modified neighbor list
+   algorithm, that uses different size bins for atom types with
+   different cutoffs and constructs neighbor lists based on those
+   cutoffs.
+
+   If atom diameters vary largely when using pair style *lj/sphere*,
```

Comment:
I see that you are discussing size ratios here.  However I don't think anyone would likely use this pair style to have particles that are (nearly) the same size, and have a variety of different cutoffs (previous paragraph).  They would just use lj/cut for that.  So I suggest making this paragraph the first and primary focus of how the user needs to be careful with this pair style.

Reply:
We may not be communicating here.  The pair style is primarily not so much different cutoffs but about having particles with different *sigma*; like a different sigma for *every* atom. Using lj/cut for that would be a pain if you have a large number of particles. The different cutoffs are a consequence of the sigmas, since it makes more sense to have the cutoff depend on sigma rather than using a fixed cutoff.